raiden-ts/tests/unit/epics/path.spec.ts

```diff
@@ -958,11 +935,10 @@ describe('PFS: pathFindServiceEpic', () => {
     setTimeout(() => action$.complete(), 10);
 
     await expect(promise).resolves.toMatchObject(
-      pathFind.failure(expect.objectContaining({ message: ErrorCodes.PFS_NO_ROUTES_FOUND }), {
-        tokenNetwork,
-        target,
-        value,
-      }),
+      pathFind.failure(
+        { message: ErrorCodes.PFS_NO_ROUTES_BETWEEN_NODES },
```

Comment:
Is this actually a PFS error? :D
I know this related to the commit before which is throwing this error. But if the nitpick-policy recognized that this is not error of the path finding **service**. :stuck_out_tongue:

Reply:
Not all PFS errors come from the remote service: PFS errors are errors on the pathFinding epics and actions functionality. So this is it.